README.md

```diff
@@ -136,9 +139,9 @@ ReactDOM.render(
 
 ### <a name="uncontrolled-mode"></a>Uncontrolled Mode ###
 
-Sometimes there is no need to handle all the component's updates, thus DevExtreme components can manage their state internally which reduces the amount of code required.
+Sometimes there is no need to handle all the component's updates, thus DevExtreme components can manage their state internally. This reduces the amount of code required.
 
-Note that if you want to specify an initial value for an option in the uncontrolled mode, you should use an appropriate property with the `default` prefix. In the example below, the `currentView` option's initial value is defined using the `defaultCurrentView` property.
+Note that if you want to specify an initial value for an option in the uncontrolled mode, use an appropriate property with the `default` prefix. In the example below, the `currentView` option's initial value is defined using the `defaultCurrentView` property.
```

Comment:
Again, why 'appropriate'?

Reply:
In the example below, the `currentView` option's initial value is defined using the `defaultCurrentView` property -> In the example below,  the `defaultCurrentView` property is used to define the `currentView` option's initial value.